src/nvme/nbft.h

```diff
@@ -0,0 +1,141 @@
+// SPDX-License-Identifier: LGPL-2.1-or-later
+/*
+ * This file is part of libnvme.
+ * Copyright (c) 2021-2022, Dell Inc. or its subsidiaries.  All Rights Reserved.
+ *
+ * Authors: Stuart Hayes <Stuart_Hayes@Dell.com>
+ *
+ */
+#ifndef _NBFT_H
+#define _NBFT_H
+
+#include <uuid/uuid.h>
+#include <ccan/list/list.h>
+
+enum primary_admin_host_flag {
+	not_indicated,
+	unselected,
+	selected,
+	reserved,
+};
+
+struct nbft_host {
+	uuid_t *id;
+	char *nqn;
+	bool host_id_configured;
```

Comment:
Code style question: now that I proposed the TP8012 enums to be public, does it make sense to provide all the respective flags as bools or better to reuse the original enums (with the need to apply bitmasks)? Thinking about possible future expansion here - it's tough to extend public structs with new elements, enums would absorb some burden.